fn_ansible_tower/apikey_permissions.txt

```diff
@@ -10,17 +10,17 @@ read_function
 #
 ## incident permissions
 #create_incident
-#read_all_incidents
+read_all_incidents
 #delete_all_incidents
 #edit_all_incidents_members
 #edit_all_incidents_owner
 #edit_all_incidents_status
 #
 ## incident fields, artifacts, attachments, milestones, datatables
-#edit_all_incidents_fields
+edit_all_incidents_fields
 #
 ## notes permissions
-#edit_all_incident_notes
+edit_all_incident_notes
```

Comment:
why is this needed?

Reply:
This updates incident notes, does this permission not required for updating notes?